src/iguana/algorithms/clas12/ZVertexFilter/Algorithm.h

```diff
@@ -26,10 +26,12 @@ namespace iguana::clas12 {
       void Stop() override;
 
       /// @action_function{scalar filter} checks if the Z Vertex is within specified bounds if pid is one for which the filter should be applied to
+      /// cuts applied to particles in FD or CD (ie not in FT)
```

Comment:
doxygen won't insert line breaks (unless forced), so add a semicolon
```suggestion
      /// @action_function{scalar filter} checks if the Z Vertex is within specified bounds if pid is one for which the filter should be applied to;
      /// cuts applied to particles in FD or CD (ie not in FT)
```